Source/ChocolateyGui/ChocolateyGui.csproj

```diff
@@ -492,6 +492,7 @@
       <DependentUpon>Settings.settings</DependentUpon>
       <DesignTimeSharedInput>True</DesignTimeSharedInput>
     </Compile>
+    <EmbeddedResource Include="Properties\Resources.nb.resx" />

```

Comment:
yeah, only the original Resource file needs it. The only time the culture dependent resource files need it if they need to be referenced directly.